angular-chart.js

```diff
@@ -135,6 +136,12 @@
           aliasVar('chartHover', 'hover');
 
           // Order of setting "watch" matter
+          function setColoursListener() {
+            // Sets and returns the variable used to cancel the watch for colours, preventing the 
+            // diagram from being created due to a change occuring while creating a diagram
+            cancel_watch_for_colours = scope.$watch('colours', resetChart, true);
+            return cancel_watch_for_colours;
```

Comment:
Yeah, actually did that on purpose, enabling any developer to set this value to an attribute of their choosing. I agree that's not required at this time